src/main/java/org/jenkinsci/plugins/workflow/cps/CpsFlowExecution.java

```diff
@@ -408,6 +408,10 @@ public int iota() {
         return iota.incrementAndGet();
     }
 
+    int currentNodeCount() {
+        return iota.get();
```

Comment:
:bug:  Gotcha here: the iota != the actual node count.  FlowStartNode is iota 2, parallel blocks skip an iota value too.  

If you call it "approximateNodeCount" with a javadoc that the real count may be slightly less I'll be happy.